packages/component/src/Localization/ar-EG.js

```diff
@@ -0,0 +1,103 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [1, 5, 24, 48, 60000, 3600000] }] */
+
+import getLocaleString from './getLocaleString';
+
+function xMinutesAgo(dateStr) {
+  const date = new Date(dateStr);
+  const dateTime = date.getTime();
+
+  if (isNaN(dateTime)) {
+    return dateStr;
+  }
+
+  const now = Date.now();
+  const deltaInMs = now - dateTime;
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+  const countofMinutesIsTen = 10; /* used for arabic translation only */
+
+  if (deltaInMinutes < 1) {
+    return 'حالا';
+  } else if (deltaInMinutes === 1) {
+    return 'منذ دقيقة';
+  } else if ((deltaInHours < 1) & (deltaInMinutes <= countofMinutesIsTen)) {
+    return `منذ ${deltaInMinutes} دقائق`;
+  } else if (deltaInHours < 1) {
+    return `منذ ${deltaInMinutes} دقيقة`;
+  } else if (deltaInHours === 1) {
+    return `منذ ساعة`;
+  } else if (deltaInHours < 5) {
+    return `منذ ${deltaInHours} ساعات`;
+  } else if (deltaInHours <= 24) {
+    return `اليوم`;
+  } else if (deltaInHours <= 48) {
+    return `أمس`;
+  }
+  return getLocaleString(date, 'ar-EG');
+}
+
+function botSaidSomething(avatarInitials, text) {
+  // return `Bot ${avatarInitials} said, ${text}`;
+  return `البوت ${avatarInitials} قال, ${text}`;
+}
+
+function downloadFileWithFileSize(downloadFileText, fileName, size) {
+  // Full text should read: "Download file <filename> of size <filesize>"
+  // return `${downloadFileText} ${fileName} of size ${size}`;
+  return `تحميل ملف ${fileName} بحجم ${size}`;
+}
+
+function uploadFileWithFileSize(fileName, size) {
+  // return `${fileName} of size ${size}`;
+  return `${fileName} بحجم ${size}`;
+}
+
+function userSaidSomething(avatarInitials, text) {
+  // return `User ${avatarInitials} said, ${text}`;
+  return `المستخدم ${avatarInitials} قال, ${text}`;
+}
+
+export default {
+  CONNECTED_NOTIFICATION: 'متصل',
+  FAILED_CONNECTION_NOTIFICATION: 'غير قادر على الاتصال.',
+  INITIAL_CONNECTION_NOTIFICATION: 'يتم التوصيل...',
```

Comment:
instead of three separate periods, please use the ellipsis code like so:
```suggestion
  INITIAL_CONNECTION_NOTIFICATION: 'يتم التوصيل…',
```